Content.Shared/Zombies/ZombieComponent.cs

```diff
@@ -13,13 +13,6 @@ namespace Content.Shared.Zombies
     [RegisterComponent, NetworkedComponent]
     public sealed class ZombieComponent : Component
     {
-        /// <summary>
-        /// The coefficient of the damage reduction applied when a zombie
-        /// attacks another zombie. longe name
-        /// </summary>
-        [ViewVariables]
-        public float OtherZombieDamageCoefficient = 0.25f;
-
```

Comment:
Why'd it be removed?

Reply:
because i removed its use from ZombieSystem.cs

Reply:
Eh? Your PR says it's only about making zombies not able to destroy AME and Gravity

Reply:
Why did you remove it exactly? is it interfering with anything on your code? You just dont like it?

Reply:
I did replace it: the original source let a zombie still make damage do another zombie using that coefficent, I made it so it can't deal damage at all.